app/models/application_record.rb

```diff
@@ -2,14 +2,22 @@ class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
   include HalApi::RepresentedModel
 
-  @@error_message_aliases = {}
-
   def self.alias_error_messages(to_field, from_field)
-    @@error_message_aliases[to_field.to_s] = from_field.to_s
+    aliases = error_message_aliases
+    aliases[to_field.to_s] = from_field.to_s
+    class_variable_set(:@@error_message_aliases, aliases)
```

Comment:
Concurrency issue?

Reply:
I don't _think_ so, since all the models get preloaded in prod?  And in any case - should be idempotent to keep adding the same keys to the same hash, right?

Prior to this, was seeing some oddness in dev.  Where calling something like `Feed.new.error_message_aliases` would change depending on which models had been loaded already.